Tween/DelegateAnimation.cs

```diff
@@ -19,6 +39,10 @@ public class DelegateAnimation<T> : AbstractAnimation where T : AbstractAnimatio
 
 		private readonly Func<T> animationCreationFunction;
 
+		// Both ScaleTime() and ChangeSpeed() are doing the same thing, so they are sharing this callback
```

Comment:
Don't compline to use `event` -- this one has no GC overhead and I am not expecting many subscriptions from the onAnimationCreated. If you use it normally you will probably never need it anyway.